### DISCUSSION TITLE
Does heads support FDE installations with detached encrypted boot headers?

### DISCUSSION THREAD

**Discussion by arrowJW** [linuxboot/heads#1669] [Q&A]
In any machine flashed with the heads firmware, can you set up a password to protect your computer at boot-up like in BIOS? That way, nobody can even use it to boot a liveboot USB device without the password.

Also, does heads support FDE installations with detached encrypted boot headers?
My ideal set up is to have my boot partition installed and encrypted in a separate USB flash drive
so it doesn't boot to my OS without it plugged in.

What other security settings does heads have which I can customize?

**Comment by tlaurion** [comment]
> In any machine flashed with the heads firmware, can you set up a password to protect your computer at boot-up like in BIOS? That way, nobody can even use it to boot a liveboot USB device without the password.

Is https://github.com/linuxboot/heads/discussions/1520 answering your question and followup question should be moved there?

> 
> Also, does heads support FDE installations with detached encrypted boot headers? My ideal set up is to have my boot partition installed and encrypted in a separate USB flash drive so it doesn't boot to my OS without it plugged in.
> 
> What other security settings does heads have which I can customize?

This should be your question here?

**Reply by arhabd** [reply]
> > Also, does heads support FDE installations with detached encrypted boot headers? My ideal set up is to have my boot partition installed and encrypted in a separate USB flash drive so it doesn't boot to my OS without it plugged in.
> > What other security settings does heads have which I can customize?
> 
> This should be your question here?

any status update on this?

**Comment by tlaurion** [comment]
I took the freedom to rename your Q&A question to your  to answer if Heads supports FDE detached header today?

The answer is no. Not as of now.

This should be the base of the question https://osresearch.net/InstallingOS/#compatibility

@arrowJW you reformulate your question maybe?
